CreatubblesAPIClient/Sources/Requests/Gallery/MyGalleriesRequest.swift

```diff
@@ -56,14 +55,13 @@ class MyGalleriesRequest: Request {
             params["per_page"] = perPage as AnyObject?
         }
 
-        switch filter {
-        case .owned:
+        if filter == GalleriesRequestFilter.owned {
```

Comment:
Why remove the switch here? If we add a new case it would remind us to update the condition.

Reply:
Good question, I'll revert it back to switch